app/controllers/proposals_controller.rb

```diff
@@ -12,6 +12,13 @@ class ProposalsController < ApplicationController
 
   def show
     @proposal = proposal.decorate
+    cookies[:detail] = params[:detail] ? params[:detail] : nil
+    @mode = cookies[:detail]
```

Comment:
I don't think this needs to be an instance var since we aren't using it in the view